go-controller/etc/ovn_k8s.conf

```diff
@@ -1,15 +1,18 @@
 [Default]
 mtu=1400
 conntrack-zone=64000
-ovn-mode=overlay
-log-path=/var/log/openvswitch/ovn-k8s-cni-overlay.log
-unix-socket=/var/run/openvswitch/ovnnb_db.sock
-cni-conf-path=/etc/cni/net.d
-cni-link-path=/opt/cni/bin/
+
+[Logging]
+logfile=/var/log/openvswitch/ovn-k8s-cni-overlay.log
+
+[CNI]
+cni-conf-dir=/etc/cni/net.d
 cni-plugin=ovn-k8s-cni-overlay
```

Comment:
Can we have a place where we can see all the possible config values? Either somewhere in README or here would be great. You gave an example in the PR description, but I don't see any other way of finding all the possible config options besides the PR description or looking through the code.

Reply:
@alinbalutoiu will do